roles/kubernetes-apps/ingress_controller/cert_manager/defaults/main.yml

```diff
@@ -7,6 +7,12 @@ cert_manager_nodeselector: {}
 cert_manager_dns_policy: "ClusterFirst"
 cert_manager_dns_config: {}
 
+## Allow http_proxy, https_proxy and no_proxy environment variables
+## Issue https://github.com/kubernetes-sigs/kubespray/issues/2933
+## Details https://github.com/kubernetes-sigs/kubespray/blob/master/docs/proxy.md
+cert_manager_http_proxy: "{{ http_proxy|d('') }}"
+cert_manager_https_proxy: "{{ https_proxy|d('') }}"
+cert_manager_no_proxy: "{{ no_proxy|d('') }}"
```

Comment:
That's a nit but style wise I think we prefer to use `default` rather than just `d`

Reply:
Ok, I'll change it to `default`